CHANGELOG.md

```diff
@@ -111,10 +146,17 @@ versions (e.g. `find_package(SEQAN3 3.0.1)` requires at least seqan3 with a vers
 
```

Comment:
In the range section:

* `seqan3/range/view` has be renamed to `seqan3/range/views`.
* namespace `seqan3::view` has been renamed to `seqan3::views`